hwtracer/src/block.rs

```diff
@@ -50,6 +50,13 @@ impl Block {
         Self::Unknown
     }
 
+    pub fn is_unknown(&self) -> bool {
```

Comment:
What does `unknown` mean here? [A doc string might be in order?]

Reply:
Fixed in c4d926c7500dda90f238e7c3dd42107d49d4ed5a

Reply:
I'm still a bit sure what "unknown" means. I guess it means "not mapped in this executable" or something like that?

Reply:
It means "an unkown virtual address range". The notion was introduced with the new ipt decoder.

It's explained in [this commit message](https://github.com/ykjit/yk/pull/585/commits/aafa6f83fce1471c8940dfad503b8a38a9baa20c) from an earlier PR (the passage starting "The shape of `Block` had to change...").

We can expand the doc string for the variant if you want.

Reply:
I think we need to explain somewhere in the code (maybe a doc string at the top of the block or wherever) what "unknown" means -- but I'm happy with wherever you might think the best place is.

Reply:
c72dec74b5adb84cf772b451fab3aa8943576628 expands the docstring and unifies usage of the variant a bit.